### PR TITLE
Random stuff

### PR DESCRIPTION
**Pull request by JFronny**
Use LibJF resource events for sound replacing and early init, generate ores completely randomly using italian words, update gradle wrapper.\
Also generates a random codename based on the version number.

### CODE REVIEW

File: src/main/java/fr/anatom3000/gwwhit/util/ItalianUtil.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package fr.anatom3000.gwwhit.util;
```
[the remark this answers is not in the document]
Comment (MattiDragon) [reply src/main/java/fr/anatom3000/gwwhit/util/ItalianUtil.java:1]: I don't static finals should be in SCREAMING_SNAKE_CASE even if they are private

**Review by ENDERZOMBI102 — CHANGES_REQUESTED** [review]
plz fix or undo

**Comment by ENDERZOMBI102** [comment]
i’ll merge it tomorrow then, in case some bug is found you’ll still have the time to fix it

**Comment by JFronny** [comment]
block_of_videotelefonia

**Pull request merged**